kafka_control_logger/Dockerfile

```diff
@@ -9,11 +9,11 @@ ENV PYTHONDONTWRITEBYTECODE 1
 ENV PYTHONUNBUFFERED 1
 
 # install dependencies
-COPY ./requirements.txt /usr/src/app/requirements.txt
+COPY ./requirements.txt ./requirements.txt
 RUN pip install --no-cache-dir -r requirements.txt
 
 # copy project
-COPY . /usr/src/app/
+COPY . .
 
 RUN chmod +x ./start.sh
 CMD ["./start.sh"]
```

Comment:
I have a question about this file `start.sh`
The content of the file is:
```sh
#!/usr/bin/env bash
python logger.py
```
That could be easily translate it to "CMD" by: `CMD ["python", "logger.py"]`
Is there any reason behind having a sh file for doing that command?

There are others `start.sh` with more lines, but again, it can be translated easily into docker, avoiding creating the layer `RUN chmod +x ./start.sh` in the image